Rakefile

```diff
@@ -14,4 +29,11 @@ if defined? RSpec
   end
 end
 
+desc "Deploy the example app to Heroku"
+namespace :deploy do
+  task :example do
+    exec 'git push heroku `git subtree split --prefix spec/example_app`:master --force'
```

Comment:
Looks like this `heroku` remote differs from what gets set up in `bin/setup`? Can we change `bin/setup` to add contributors to the correct heroku app, with a remote name that matches this?

Also, I've been preferring the `$(shell_command)` format, because:
1. It differentiates the start and end of a command
2. It's copy-and-pastable into Slack code snippets :smile: